jsparty/js-party-121.md

```diff
@@ -154,7 +154,7 @@ One thing -- when we do this kind of projects, we use a concept which is called
 
 **Kevin Ball:** Oh, interesting. So are there situations in which you have micro frontends that aren't tied into microservices?
 
-**Michael Geers:** I've also talked to people who are -- while we are using the term micro frontends, and we are practicing it, we also associate this system with a team, and we try to do the team as cross-functional as possible... But you could also use the composition techniques as \[unintelligible 00:29:30.25\] for example the application shell, just in the frontend; just doing a classical backend, for example, but having multiple frontend teams sitting on the same backend or GraphQL data or whatever, and just using the micro frontends techniques to assemble pages and distribute frontend work to multiple teams. This is also something that's quite possible, but it's not the thing we are doing.
+**Michael Geers:** I've also talked to people who are -- while we are using the term micro frontends, and we are practicing it, we also associate this system with a team, and we try to do the team as cross-functional as possible... But you could also use the composition techniques as single spot approach for example the application shell, just in the frontend; just doing a classical backend, for example, but having multiple frontend teams sitting on the same backend or GraphQL data or whatever, and just using the micro frontends techniques to assemble pages and distribute frontend work to multiple teams. This is also something that's quite possible, but it's not the thing we are doing.
```

Comment:
Michael and KBall mention `single-spa` elsewhere in the episode, pretty sure this is another usage of that:
```suggestion
**Michael Geers:** I've also talked to people who are -- while we are using the term micro frontends, and we are practicing it, we also associate this system with a team, and we try to do the team as cross-functional as possible... But you could also use the composition techniques as single-spa approach for example the application shell, just in the frontend; just doing a classical backend, for example, but having multiple frontend teams sitting on the same backend or GraphQL data or whatever, and just using the micro frontends techniques to assemble pages and distribute frontend work to multiple teams. This is also something that's quite possible, but it's not the thing we are doing.
```